src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -614,9 +614,10 @@ private extension URLRequest {
 }
 
 private extension ENExposureConfiguration {
-	convenience init(from riskscoreParameters: SAP_RiskScoreParameters) throws {
+	convenience init(from riskscoreParameters: SAP_RiskScoreParameters, minRiskScore: Int32) throws {
 		self.init()
-		// We are intentionally not setting minimumRiskScore.
+		minimumRiskScore = UInt8(minRiskScore)
```

Comment:
why do we accept Int32 as parameter when we immediately convert it to UInt8?

Reply:
Good question. To be honest, I don't know. The "minRiskScore" property from the configuration was already there with type Int32. What can we do in this case?

Reply:
Added clamping to the type conversion, so in case minRiskScore fall below or exceeds the range of values of UInt8, it will be clamped to 0 or 255.

Reply:
that seems like a good workaround!